packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/DeviceAuthenticityOptOutModal.tsx

```diff
@@ -1,36 +1,56 @@
-import { Column, Paragraph } from '@trezor/components';
-import { Translation } from 'src/components/suite';
+import { useState } from 'react';
+
+import { Column, NewModal, Card, Banner } from '@trezor/components';
+import { Translation, CheckItem } from 'src/components/suite';
 import { deviceAuthenticityOptOut } from 'src/actions/suite/suiteActions';
 import { useDispatch } from 'src/hooks/suite';
-import { OptOutModal } from './OptOutModal';
+import { spacings } from '@trezor/theme';
 
 type DeviceAuthenticityOptOutModalProps = {
     onCancel: () => void;
 };
 
 export const DeviceAuthenticityOptOutModal = ({ onCancel }: DeviceAuthenticityOptOutModalProps) => {
+    const [isConfirmed, setIsConfirmed] = useState(false);
     const dispatch = useDispatch();
 
+    const handleTurningOffRevisionCheck = () => {
+        dispatch(deviceAuthenticityOptOut(true));
+        onCancel();
+    };
+
     return (
-        <OptOutModal
-            headingKey="TR_DEVICE_AUTHENTICITY_OPT_OUT_MODAL_HEADING"
-            checkboxLabelKey="TR_DEVICE_AUTHENTICITY_OPT_OUT_MODAL_CHECKBOX_TITLE"
-            confirmKey="TR_DEVICE_AUTHENTICITY_OPT_OUT_MODAL_BUTTON"
-            data-testid="@device-authenticity"
-            onChange={({ isDisabled }) => dispatch(deviceAuthenticityOptOut(isDisabled))}
+        <NewModal
             onCancel={onCancel}
+            heading={<Translation id="TR_DEVICE_AUTHENTICITY_OPT_OUT_TITLE" />}
+            description={<Translation id="TR_DEVICE_AUTHENTICITY_OPT_OUT_MODAL_DESCRIPTION_3" />}
+            bottomContent={
+                <NewModal.Button
+                    onClick={handleTurningOffRevisionCheck}
+                    isDisabled={!isConfirmed}
+                    data-testid="@device-authenticity/opt-out-button"
+                >
+                    <Translation id="TR_DEVICE_AUTHENTICITY_OPT_OUT_MODAL_BUTTON" />
+                </NewModal.Button>
+            }
+            variant="warning"
         >
-            <Column gap={16}>
-                <Paragraph>
+            <Column gap={spacings.sm}>
+                <Banner icon="questionFilled">
                     <Translation id="TR_DEVICE_AUTHENTICITY_OPT_OUT_MODAL_DESCRIPTION_1" />
-                </Paragraph>
-                <Paragraph>
+                </Banner>
+                <Banner icon="warningFilled">
                     <Translation id="TR_DEVICE_AUTHENTICITY_OPT_OUT_MODAL_DESCRIPTION_2" />
-                </Paragraph>
-                <Paragraph>
-                    <Translation id="TR_DEVICE_AUTHENTICITY_OPT_OUT_MODAL_DESCRIPTION_3" />
-                </Paragraph>
+                </Banner>
             </Column>
-        </OptOutModal>
+            <Card margin={{ top: spacings.lg }}>
+                <CheckItem
+                    title={<Translation id="TR_DEVICE_AUTHENTICITY_OPT_OUT_MODAL_CHECKBOX_TITLE" />}
+                    isChecked={isConfirmed}
+                    onClick={() => setIsConfirmed(!isConfirmed)}
```

Comment:
Nit: in this simple case it doesn't matter, but generally setState should not rely on itself to be reliable. Rather reference previous value:
```suggestion
                    onClick={() => setIsConfirmed(prev => !prev)}
```